operator/src/main/java/oracle/kubernetes/operator/helpers/BasePodStepContext.java

```diff
@@ -160,6 +162,16 @@ protected List<V1EnvVar> createEnv(AuxiliaryImage auxiliaryImage, String name) {
     return vars;
   }
 
+  protected V1ResourceRequirements createResources() {
+    V1ResourceRequirements resources = getServerSpec().getResources();
+    V1ResourceRequirements resourceRequirements = null;
+    if (!resources.getLimits().isEmpty() || !resources.getRequests().isEmpty()) {
```

Comment:
could getLImits() or getRequests() return null?

Reply:
Changed in [b8ae10d](https://github.com/oracle/weblogic-kubernetes-operator/pull/3028/commits/b8ae10deb3a77aaa0144000f69b3dfdc73ebae5e).